pipeline/tier_executor.groovy

```diff
@@ -15,66 +15,66 @@ def buildType
 def final_stage = false
 def run_type
 def nodeName = "centos-7"
-def tags = "${params.tags}" ?: ""
-tags_list = tags.split(',') as List
-if ("ibmc" in tags_list){
-    nodeName = "agent-01"
-}
+def tags = ""
+
+def branch='origin/master'
+def repo='https://github.com/red-hat-storage/cephci.git'
 
-def branch
-def repo
-if (params.gitbranch){branch = "*/${params.gitbranch}"}
-else{branch='origin/master'}
+if (params.containsKey('gitbranch')){
+branch=params.gitbranch}
```

Comment:
Just a minor change. Can we please correct the indentation in these if conditions.

Reply:
Done